packages/kuma-gui/src/app/data-planes/routes.ts

```diff
@@ -66,26 +66,7 @@ export const routes = () => {
         props: () => ({
           routeName: `${fullPrefix}data-plane-summary-view`,
         }),
-        redirect: { name: `${fullPrefix}data-plane-summary-overview-view` },
         component: () => import('@/app/data-planes/views/DataPlaneSummaryView.vue'),
-        children: [
-          {
-            path: 'summary-overview',
```

Comment:
So we avoid this for the moment also 🎉 

Good that we avoid it here, that said I think we are going to have to change things slightly in the future so we can avoid such URL clashes totally

(the reason we where using `summary-overview` as a solution was in explained in https://github.com/kumahq/kuma-gui/pull/3209#discussion_r1851696117 and we have to do this because of the unfortunate way our URLs are currently)

Reply:
Yes, exactly 😄 👍

Reply:
I think this is a good PR to cross reference here https://github.com/kumahq/kuma-gui/pull/1459

Not sure if its super clear from the conversation there but I was always very keen to use plurals for listings/summaries and singular for detail pages:

Something like:

- `/dataplanes` <- listing no summary
- `/dataplanes/dataplane/my-dataplane/overview` <- listing with summary overview for my-dataplane
- `/dataplane/my-dataplane/overview` <- detail page overview for my-dataplane